trame_server/core.py

```diff
@@ -486,12 +490,9 @@ def start(
         self._running_stage = 1
         task = CoreServer.server_start(
             options,
-            **{  # Do a proper merging/override
-                **kwargs,
-                "disableLogging": disable_logging,
-                "backend": backend,
-                "exec_mode": exec_mode,
-            },
```

Comment:
I want to keep the **kwargs merge in case wslink add new args. Options are indeed different. So having host extracted make sense.

Reply:
I've update this to pass `**kwargs` without this merge logic... effectively passing unused kwargs for future compatibility and no current use

Reply:
By doing so, if someone pass `disableLogging` as a kwarg, you get an exeception.

Reply:
Okay, I went back to the way it was (mostly)